pages/api/studentPortal/interviews/LuckyInterviewData.ts

```diff
@@ -0,0 +1,138 @@
+import { Offer } from "../../../../components/resources/jobTracker/JobTrackerComponent";
+// Lucky H Interview Data 
+export const LuckyInterviewData: Offer[]=[
+    {
+      "Company": "Robinhood",
+      "companyUrl": "https://www.robinhood.com/",
+      "Location": "Toronto",
+      "Role": "Mobile Developer",
+      "Time Spent": "March 2024",
+      "timeSpentHours": 10,
+      "Result": "Failed First Technical Round",
+      "Base": 115000,
+      "opportunitySource": "Livecoding - List of instruments with filtering",
```

Comment:
I think we should keep opportunity score and time spent. I agree adding what you were asked to do and having the timeline is helpful but I would add those as additional properties and just optionally render them on your profile view. 

It's confusing to see the property name not match the value in this data. 

Add your custom properties instead of replacing existing ones. Your custom properties can be optional so you don't affect the other offer table components

Reply:
Right makes sense, sounds good to me.